2_process/src/tally_site_obs.R

```diff
@@ -9,3 +9,7 @@ tally_site_obs <- function(site_data) {
     group_by(Site, State, Year) %>%
     summarize(NumObs = length(which(!is.na(Value))), .groups = "keep")
 }
+
+combine_obs_tallies <- function(...){
+  vctrs::vec_c(...)
```

Comment:
I think that the `tally` targets are data.frames, so you might need to switch this to something that can bind data.frames, rather than vectors (maybe `dplyr::bind_rows()`?)

Reply:
Oh, I was wondering about that. That `bind_rows()` is giving me an error. 

Error : Problem with `mutate()` column `NumObsBin`.
i `NumObsBin = cut(NumObs, breaks = c(0, 100, 200, 300, 364, 366))`.
x object 'NumObs' not found

basically `obs_tallies` is empty.

When I put `browser()` in the `tally_site_obs` function, it doesn't step inside. Hmmm. Any ideas why? is it because we're doing the branching.

Reply:
You should be able to still use `browser()` even if you are using branching. Did you use `tar_make(callr_function = NULL)` to try to kick off the pipeline after you added browser?

Reply:
ahhhh, I forgot to set that to NULL. TY!!!!